tests/test_hypersphere.py

```diff
@@ -396,6 +396,34 @@ def test_mean_and_belongs(self):
         result = self.metric.mean([point_a, point_b, point_c])
         self.assertTrue(self.space.belongs(result))
 
+    def test_random_vonMisesFisher(self):
+        """
+        Check that the maximum likelihood estimates of the mean and
+        concentration parameter are close to the real values for a large
+        sample size.
+        """
+        dim = 2
+        n_points = 1000000
+        sphere = Hypersphere(dim)
+
+        # check mean value for concentrated distribution
+        K = 10
+        points = sphere.random_vonMisesFisher(K, n_points)
+        S = gs.sum(points, axis=0)
+        mu_hat = S/gs.linalg.norm(S)
```

Comment:
could we write `mean` instead of `mu`? the code is more readable if the variable can be interpreted in english without refering to notation convention (especially for future contributors).

Reply:
or `mean_estimate`?